README.md

```diff
@@ -98,10 +97,10 @@ and Drop component to a lesson, then click the `EDIT` button.
 ![Edit view](https://raw.githubusercontent.com/edx-solutions/xblock-drag-and-drop-v2/c955a38dc3a1aaf609c586d293ce19b282e11ffd/doc/img/edit-view.png)
```

Comment:
@haikuginger done. Just in case no one told you that and onboarding doc does not mention it we have at least two conventions:

* Reviewer formally approves PR by submitting :+1: (+1) emoji in comment.
* We've established a "conditional +1" thing, which means that the remaining notes are simple and straightforward enough that you trust PR owner to fix them and do not require additional review from you. This helps in cases where timezones are significantly different (like between us) by saving another round-trip.

Example 1: "conditional +1 provided typos are fixed"
Example 2: this could have been a conditional too :)

Also, just FYI, we have a notion of "nit" (a shorthand for nitpick I believe) - those are minor improvements that are still useful, but could be skipped in case of lack of time, too complex fix, etc.; in short - PR can be merged with nits. We tend to do them most of the time anyway.